changelog/fragments/align-kb.yaml

```
@@ -0,0 +1,124 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Go-based operators, migrate your project to use the project version stable.
+    kind: "change"
+    breaking: true
+    migration:
+      header: For Go-based operators, migrate your project to use the project version stable.
+      body: >
+        The PROJECT version config file represents the project configuration. It reach the maturate stability and it will
+        store the data used to do the scaffolds. The motivation for this behaviour is to allow tools and helpers in the future such as to make easier the process to update the projects to use upper versions.
+        More info: TBD. //TODO: add link for docs
+  - description: >
+      For Ansible/Helm-based operators, usage of project config stable version.
+    kind: "change"
+    breaking: true
+    migration:
+      header: For Ansible/Helm-based operators, update the PROJECT file.
+      body: >
+        Update the PROJECT file by replacing the `version: 3-alpha` with `version: "3"` and then, for each API scaffolded in your project:
+        - Add the `- api.crdVersion: <crdVersion>` and `api.namespaced: true`.
+        - Add the resource `domain` which will be the same value of the project domain unless it is an external type
+
+        Ensure that the PROJECT file results are like:
+
+        ```yml
+        domain: example.com
+        projectName: memcached-operator
+        resources:
+        - api:
+            crdVersion: v1
+            namespaced: true
+          domain: example.com
+          group: cache
+          kind: Memcached
+          version: v1alpha1
+        version: "3"
+        ```
+  - description: >
+      Remove spaces from maker comments to make them machine-readable. More info: [#kubernetes-sigs/kubebuilder#1868](https://github.com/kubernetes-sigs/kubebuilder/pull/1868).
+    kind: "bugfix"
+    breaking: false
+    migration:
+      header: Update the comments makers of your project to make them machibe redable.
+      body: >
+        Update your project files removing the maker's comments spaces as follows:
+        - Replace `# +kubebuilder` with `#+kubebuilder`
+        - Replace `// +groupName` with `//+groupName`
+        - (For Go-based operators only) Replace `// +operator-sdk` with `//+operator-sdk`
+  - description: >
+      Add support for markers for files with the `yml` extension. More info: [#kubernetes-sigs/kubebuilder#1907](https://github.com/kubernetes-sigs/kubebuilder/pull/1907).
+    kind: "addition"
+    breaking: false
+  - description: >
+      For Go-based operators using go/v3 plugin only, add the `--force` option to the `create webhook` command. More info: [#kubernetes-sigs/kubebuilder#1903](https://github.com/kubernetes-sigs/kubebuilder/pull/1903).
```

If this only affects go/v3, then use the plugin key
```suggestion
      (go/v3) Added the `--force` option to the `create webhook` command. More info: [#kubernetes-sigs/kubebuilder#1903](https://github.com/kubernetes-sigs/kubebuilder/pull/1903).
```